// Copyright (c) 2023 Boston Dynamics, Inc.  All rights reserved.
//
// Downloading, reproducing, distributing or otherwise using the SDK Software
// is subject to the terms and conditions of the Boston Dynamics Software
// Development Kit License (20191101-BDSDK-SL).

syntax = "proto3";

package bosdyn.api;

option java_outer_classname = "PayloadAccessServiceProto";

import "bosdyn/api/payload.proto";

// This service provides a way to query for the currently-registered payloads.
service PayloadService {
    // List all payloads the robot knows about.
    rpc ListPayloads(ListPayloadsRequest) returns (ListPayloadsResponse);
}

